debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4

```diff
@@ -432,7 +432,7 @@ tableOption
     | ROW_FORMAT '='?
         rowFormat=(
           DEFAULT | DYNAMIC | FIXED | COMPRESSED
-          | REDUNDANT | COMPACT
+          | REDUNDANT | COMPACT | ID
```

Comment:
LGTM, thanks. Could you please place it on an extra line and add a comment it is for TokuDB only. In the similar way you do fro MariaDB extensions.

Reply:
I actually tried using explicit tokens for `TOKUDB_XXXX` formats and this seemed to confuse the parser when running the test, `MySqlAntlrDdlParserTest#shouldParseThirdPartyStorageEngine` where the create table statement is:
```
CREATE TABLE escaped_foo ( ... ) engine=TokuDB `compression`=tokudb_zlib;
```
The only way I was able to get the test to pass was to use `ID` rather than include the individual `TOKUDB_XXX` formats.  Do you have any suggestions for that which I couldn't spot?

Reply:
I see, I remeber it now! Please take a look at `engineName` rule. We included the ID and subsequent tokens to cover for arbitrary values to cover for non-standard engines.

Could you please verify if using of quoted string in `COMPRESSSION` is valid for TokuDB? If yes the rule should be xpanded in the same way as the engine one and we are good to go for all future values.

Reply:
I just found this [reference](https://github.com/percona/tokudb-engine/wiki/Compression) and it does seem that the value that is associated with `COMPRESSION` can be single-quoted but I'm not finding any reference that indicates it can be double quoted.

Reply:
ok, let's keep it as is then. We can return back to it when the need arises.